src/main/java/com/gtnewhorizons/gravisuiteneo/GraviSuiteNeoRegistry.java

```diff
@@ -90,7 +90,7 @@ private static void initFuelmap() {
             }
             fluids.put(fluid, fuelValue);
         }
-        if (Loader.isModLoaded("gregtech")) {
+        if (Loader.isModLoaded("dreamcraft")) {
```

Comment:
Use
```suggestion
        if (Loader.isModLoaded("gregtech") && !Loader.isModLoaded(“gregapi")) {
```
Instead, to check for GT6, since gregapi is a part of that mod.